runners/spark/src/main/java/org/apache/beam/runners/spark/util/SparkSideInputReader.java

```diff
@@ -61,6 +61,7 @@ public SparkSideInputReader(Map<TupleTag<?>, KV<WindowingStrategy<?, ?>, Broadca
     //--- match the appropriate sideInput window.
     // a tag will point to all matching sideInputs, that is all windows.
     // now that we've obtained the appropriate sideInputWindow, all that's left is to filter by it.
+    @SuppressWarnings("unchecked")
```

Comment:
Unchecked cast: 'java.lang.Object' to 'java.lang.Iterable<org.apache.beam.sdk.util.WindowedValue<?>>' less... (⌘F1) 
Signals places where an unchecked warning is issued by the compiler, for example:

  void f(HashMap map) {
    map.put("key", "value");
  }